delta/plugins/composite-views/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/migration/MigrateCompositeViewsSuite.scala

```diff
@@ -0,0 +1,176 @@
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.model.{CompositeViewEvent, CompositeViewState, CompositeViewValue}
+import ch.epfl.bluebrain.nexus.delta.rdf.syntax.iriStringContextSyntax
+import ch.epfl.bluebrain.nexus.delta.sdk.crypto.Crypto
+import ch.epfl.bluebrain.nexus.delta.sourcing.Transactors
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.{ProjectRef, Tag}
+import ch.epfl.bluebrain.nexus.delta.sourcing.implicits._
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.Tag.UserTag
+import doobie.postgres.implicits._
+import ch.epfl.bluebrain.nexus.delta.sourcing.postgres.Doobie
+import ch.epfl.bluebrain.nexus.testkit.bio.BioSuite
+import doobie.Get
+import munit.{AnyFixture, Location}
+import doobie.implicits._
+import io.circe.syntax.EncoderOps
+import io.circe.JsonObject
+import monix.bio.IO
+
+import java.time.Instant
+
+class MigrateCompositeViewsSuite extends BioSuite with Doobie.Fixture with ClasspathResourceUtils {
+
+  private val proj = ProjectRef.unsafe("myorg", "myproj")
+
+  override def munitFixtures: Seq[AnyFixture[_]] = List(doobie)
+
+  implicit private lazy val xas: Transactors = doobie()
+
+  private val crypto: Crypto                     = Crypto("changeme", "salt")
+  private val eventSerializer                    = CompositeViewEvent.serializer(crypto)
+  implicit val eventGet: Get[CompositeViewEvent] = eventSerializer.getValue
+  private val stateSerializer                    = CompositeViewState.serializer(crypto)
+  implicit val stateGet: Get[CompositeViewState] = stateSerializer.getValue
+
+  private def assertMigratedValue(value: CompositeViewValue, rev: Int)(implicit loc: Location): Unit = {
+    assertEquals(value.sourceIndexingRev, rev)
+    value.projections.toNel.toList.foreach { case (_, projection) =>
+      assertEquals(projection.indexingRev, rev)
+    }
+  }
+
+  test("Insert states and events and run migration") {
+    for {
+      _ <- xas.initPartitions(proj)
+      // Events to migrate
+      _ <- loadEvent("migration/event-created.json")
+      _ <- loadEvent("migration/event-updated.json")
+      // Events already migrated
+      _ <- loadEvent("composite-views/database/named-view-created.json")
+      _ <- loadEvent("composite-views/database/named-view-updated.json")
+      // States to migrate
+      _ <- loadState(Tag.latest, "migration/state.json")
+      _ <- loadState(UserTag.unsafe("v1.0"), "migration/state.json")
+      _ <- loadState(Tag.latest, "composite-views/database/view-state.json")
+      _ <- loadState(UserTag.unsafe("v1.0"), "composite-views/database/view-state.json")
+      _ <- eventsToMigrate(xas).map(_.assertSize(2))
+      _ <- statesToMigrate(xas).map(_.assertSize(2))
+      _ <- new MigrateCompositeViews(xas).run.assert((2, 2))
+      // Making sure all has been updated
+      _ <- eventsToMigrate(xas).map(_.assertSize(0))
+      _ <- statesToMigrate(xas).map(_.assertSize(0))
+    } yield ()
+  }
```

Comment:
This shouldn't be a test, it's just setup

Reply:
It is not only setup, it also run the migration.